daemon/create.go

```diff
@@ -14,26 +14,35 @@ import (
 	"github.com/opencontainers/runc/libcontainer/label"
 )
 
+// ContainerCreate is the parameter set to ContainerCreate()
+type ContainerCreate struct {
+	Name            string
+	Config          *runconfig.Config
+	HostConfig      *runconfig.HostConfig
+	AdjustCPUShares bool
+	FromBuilder     bool
```

Comment:
Right, but we shouldn't be parsing config.Volumes anymore, right?
Builder would only use config.Volumes.